modules/nlu/src/backend/pos-tagger.ts

```diff
@@ -1,10 +1,40 @@
 import * as sdk from 'botpress/sdk'
 import path from 'path'
-import yn from 'yn'
 
 import { isSpace, SPACE } from './tools/token-utils'
 
-const USE_POS = yn(process.env.BP_EXPERIMENTAL_NLU_POS)
+export function isPOSAvailable(lang: string): boolean {
+  // TODO check that language is part of supported languages once we support more
+  return lang === 'en'
+}
+type ElementType<T extends ReadonlyArray<unknown>> = T extends ReadonlyArray<infer ElementType> ? ElementType : never
+
+export type POS_CLASS = ElementType<typeof POS_CLASSES>
```

Comment:
I suggest using the standard naming for interface (ex: PosClass), I was expecting a variable being all upper cases